Lib/test/test_capi/test_exceptions.py

```diff
@@ -6,7 +6,7 @@
 import textwrap
 
 from test import support
-from test.support import import_helper
+from test.support import import_helper, force_not_colorized
```

Comment:
Maybe directly use `support.force_not_colorized`? it's less diff (I also think it's better when we don't mix module imports and attribute imports)